src/DssSpell.sol

```diff
@@ -75,334 +60,240 @@ contract DssSpellAction is DssAction {
+            // Set parameter l2BridgeInstance.govRelay: (L2GovernanceRelay: 0xdD0BCc201C9E47c6F6eE68E4dB05b652Bb6aC255)
+            govRelay: BASE_GOV_RELAY_L2,
+            // Set parameter l2BridgeInstance.bridge: (ERC1967Proxy: 0xee44cdb68D618d58F75d9fe0818B640BD7B8A7B7)
+            bridge: BASE_TOKEN_BRIDGE_L2,
+            // Set parameter l2BridgeInstance.bridgeImp: (L2TokenBridge: 0x289A37BE5D6CCeF7A8f2b90535B3BB6bD3905f72)
+            bridgeImp: BASE_TOKEN_BRIDGE_IMP_L2,
+            // Set parameter l2BridgeInstance.spell: (L2TokenBridgeSpell: 0x6f29C3A29A3F056A71FB0714551C8D3547268D62)
+            spell: SPELL_L2
+        });
+
+        // Array with mainnet tokens
+        address[] memory l1Tokens = new address[](2);
+        l1Tokens[0] = USDS;
+        l1Tokens[1] = SUSDS;
+
+        // Array with Base tokens
+        address[] memory l2Tokens = new address[](2);
+        l2Tokens[0] = USDS_L2;
+        l2Tokens[1] = SUSDS_L2;
+
+        // Max withdrawals for Base tokens
+        uint256[] memory maxWithdrawals = new uint256[](2);
+        maxWithdrawals[0] = type(uint256).max;
+        maxWithdrawals[1] = type(uint256).max;
+
+        // Set cfg with the following parameters:
+        BridgesConfig memory bridgeCfg = BridgesConfig({
+            // Set parameter cfg.l1Messenger: (l1messenger 0x866E82a600A1414e583f7F13623F1aC5d58b0Afa)
+            l1Messenger: MESSENGER,
+            // Set parameter cfg.l2Messenger: (l2messenger 0x4200000000000000000000000000000000000007)
+            l2Messenger: MESSENGER_L2,
+            // Set parameter cfg.l1Tokens: (USDS, SUSDS on mainnet)
+            l1Tokens: l1Tokens,
+            // Set parameter cfg.l2Tokens: (USDS: 0x820C137fa70C8691f0e44Dc420a5e53c168921Dc, sUSDS: 0x5875eEE11Cf8398102FdAd704C9E96607675467a on Base chain)
+            l2Tokens: l2Tokens,
+            // Set parameter cfg.maxWithdraws: (type(uint256).max for each token)
+            maxWithdraws: maxWithdrawals,
+            // Set parameter cfg.minGasLimit: (500,000)
+            minGasLimit: 500_000,
+            // Set parameter cfg.govRelayCLKey: (chainlog key for govRelay -> BASE_GOV_RELAY)
+            govRelayCLKey: "BASE_GOV_RELAY",
+            // Set parameter cfg.escrowCLKey: (chainlog key for Escrow -> BASE_ESCROW)
+            escrowCLKey: "BASE_ESCROW",
+            // Set parameter cfg.l1BridgeCLKey: (chainlog key for L1TokenBridge -> BASE_TOKEN_BRIDGE)
+            l1BridgeCLKey: "BASE_TOKEN_BRIDGE",
+            // Set parameter cfg.l1BridgeImpCLKey: (chainlog key for L1TokenBridgeImp -> BASE_TOKEN_BRIDGE_IMP)
+            l1BridgeImpCLKey: "BASE_TOKEN_BRIDGE_IMP"
+        });
+
+        // Init Base Token Bridge for USDS, sUSDS
+        TokenBridgeInit.initBridges(dss, l1BridgeInstance, l2BridgeInstance, bridgeCfg);
+
+
+        // ---------- Init Allocator System for Spark Subdao Proxy ----------
+        // Forum: https://forum.sky.money/t/spell-contents-2024-10-31/25421/
+        //
```

Comment:
`Init shared components for Allocator System be calling AllocatorInit.initShared` comment is missing